lib/reversetunnel/remotesite.go

```diff
@@ -828,6 +828,13 @@ func (s *remoteSite) dialAndForward(params reversetunnelclient.DialParams) (_ ne
 		return nil, trace.Wrap(err)
 	}
 
+	// If the node is unknown (was directly dialed) tell the forwarding
+	// server it isn't an agentless node so config checks pass.
+	// params.TargetServer will ensure the node is not treated as a
+	// Teleport node in this case.
+	// DELETE in 15.0.0
+	isAgentlessNode := params.IsAgentlessNode && !params.IsUnknownNode
```

Comment:
Can you extract this condition to a function to remove repetition between here and `lib/reversetunnel/localsite.go`?

Reply:
Done